### DISCUSSION TITLE
Unable to save updates to USS files

### DISCUSSION THREAD

**Discussion by rsjrny** [zowe/zowe-explorer-vscode#2035] [Q&A] [answered]
Is there a setting I must select to have my updates to USS files saved back to USS?

I am able to edit a file from USS without issue and the first save seems to work. If I make other changes on the same file they are not saved back to USS 

I am on VS Code:
Version: 1.74.0 (user setup)
Commit: 5235c6bb189b60b01b1f49062f4ffa42384f8c91
Date: 2022-12-05T16:38:16.075Z
Electron: 19.1.8
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

and Zowe Explorer: v2.4.1

**Comment by traeok** [comment, marked as answer]
Hi @rsjrny ,

Sorry for the delay. Are there any error messages or dialogs that appear on all subsequent saves? Or are these save operations made without error, but yet the file has not been saved with the newest changes?

In the meantime, these are the steps I will attempt to try reproducing the issue:

1. Open a USS file in Zowe Explorer
2. Make changes to said file
3. Save file for the first time
4. Make new changes to file in editor, and try saving again
5. Note that subsequent changes to USS file are not saved on mainframe

Finally, do you still encounter this issue with Zowe Explorer v2.8.0? These changes might have been resolved since version 2.4.1 of Zowe Explorer.

Thanks in advance!

**Reply by rsjrny** [reply]
Hi Trae,

This was fixed some time ago with one of the updates.

Thanks.

**Reply by traeok** [reply]
Great! Glad to hear it was resolved. Marking this as the answer and closing as resolved.
